src/pet.js

```diff
@@ -0,0 +1,59 @@
+const MAXIMUM_FITNESS = 10;
+const AGE_AT_BIRTH = 0;
+const NOT_HUNGRY = 0;
+const isDeadError = new Error('Sorry, your pet has gone to live on the big farm in the sky');
+
+function Pet(name) {
+    this.name = name,
+    this.age = AGE_AT_BIRTH,
+    this.hunger = NOT_HUNGRY,
+    this.fitness = MAXIMUM_FITNESS,
+    this.children = []
+};
+
+Pet.prototype = {
+    get isAlive() {
+        return this.fitness > 0 && this.hunger < 10 && this.age < 30
+    },
+    growUp() {
+        if (!this.isAlive) {
+            throw isDeadError
+        }
+        this.age += 1,
+        this.hunger += 5,
+        this.fitness -= 3
```

Comment:
as above, any time you use a number that looks arbitrary, you should be assigning that number to a variable with a descriptive name about what that number refers to